tests/lib/rule-tester/rule-tester.js

```diff
@@ -463,6 +463,92 @@ describe("RuleTester", () => {
         });
     });
 
+    describe("before", () => {
+        const ruleName = "no-var";
+        const rule = require("../../fixtures/testers/rule-tester/no-var");
+
+        it("should be called when a function is assigned", () => {
```

Comment:
three tests are similar for both hooks, so that I'd like to use something similar to `.each()` approach, but looks like Mocha does not have it, so will attempt to use `for..of` or `Array::forEach()`

Reply:
addressed in 94ef69259